src/scss/_ansi.scss

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+/* ansi text */
+.ansi-bold {
+  font-weight: bold;
+}
+
+.ansi-faint {
+  // cannot use sass variables in functions
+  color: darken(hsl(0, 0%, 98%), 50%);
+}
+
+.ansi-framed {
+  // cannot use sass variables in functions
+  background-color: darken(hsl(0, 0%, 98%), 70%);
+  border: 1px solid darken(hsl(0, 0%, 98%), 50%);
+  border-radius: 5px;
+}
+
+.ansi-black-fg {
+  color: var(--color-coal-light);
+}
+
+.ansi-red-fg {
```

Comment:
these are the juice of the PR
I tried to re-use as many of the default colors as I could for the ANSI colors